tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": "@eggjs/tsconfig",
+  "compilerOptions": {
+    "target": "es2022",
+    "module": "nodenext",
+    "moduleResolution": "nodenext"
+  },
+  "include": [
```

Comment:
no need to set include and exclude settings.

Reply:
have deleted